lib/models/player.rb

```diff
@@ -0,0 +1,16 @@
+module RubyConnect
+  class Player
+    attr_reader :game, :color
+
+    def initialize(info = {})
+      @name         = info[:name]
+      @phone_number = info[:phone_number]
+      @color        = info[:color]
+      @game         = info[:game]
```

Comment:
Please, help me think about this. Given a game, we need to retrieve two related players. And one player can play many games at the same time. So, should a player `has_many :games`?